trieste/acquisition/rule.py

```diff
@@ -63,19 +57,19 @@ def acquire(
 
         **Type hints:**
           - The global search space must be a :class:`~trieste.space.SearchSpace`. The exact type
-            of :class:`~trieste.space.SearchSpace` depends on the specific
-            :class:`AcquisitionRule`.
+                of :class:`~trieste.space.SearchSpace` depends on the specific
+                :class:`AcquisitionRule`.
```

Comment:
why these?

Reply:
black did it

Reply:
that's odd. black shouldn't touch docstrings

Reply:
ok that's not really true having checked, but it's never modified these lines before and the version of black is pinned. It also look wrong.

Reply:
how are you running black?

Reply:
just from command line. as "black ."

Reply:
also makes this PR appear very large unfortunately!

Reply:
do you mind reverting them?

Reply:
Yep, doing now